src/discord/staffcommands.py

```diff
@@ -366,11 +371,10 @@ async def confirm(self, interaction: discord.Interaction, member: discord.Member
                 f"channels and send messages in them. :tada: "
             )
 
-    @app_commands.context_menu(name="Confirm User")
     @app_commands.checks.has_any_role(ROLE_STAFF, ROLE_VIP)
     @app_commands.default_permissions(manage_roles=True)
     @app_commands.guilds(SLASH_COMMAND_GUILDS)
-    async def confirm_user(self, interaction, member: discord.Member):
+    async def confirm_user(self, interaction: discord.Interaction, member: discord.Member):
```

Comment:
pretty sure discord.py context menus must be functions or staticmethods as they can only have 2 params, one being interaction and then the other either being discord.Message or discord.Member